src/utils/data.js

```diff
@@ -137,8 +136,7 @@ export function clone(jsonObject) {
   return JSON.parse(JSON.stringify(jsonObject));
 }
 
-export async function fetchIndexPathsData({ currentLocale }) {
-  const locale = currentLocale === defaultLocale ? '' : currentLocale;
-  const path = new URL(`${pathJoin([baseUrl, 'index/', locale, 'index.json'])}`, window.location.href);
+export async function fetchIndexPathsData({ slug }) {
```

Comment:
I know this was already merged, but it might be good to rename this in a future PR to make it more clear that the slug here refers to the slug for a locale and not something else, like the technology name or whatever.